scss/components/_card.scss

```diff
@@ -38,13 +38,21 @@ $card-padding: $global-padding !default;
 /// @type number
 $card-margin: $global-margin !default;
 
+/// Default columns in a `.card-columns` wrapper.
+/// @type number
+$card-columns-count: 3 !default;
+
+/// Default gutter between cards in a `.card-columns` wrapper.
+/// @type number
+$card-columns-gap: $card-margin !default;
+
 /// Adds styles for a card container.
-/// @param {Color} $background - Background color of the card.
-/// @param {Color} $color - font color of the card.
-/// @param {Number} $margin - Bottom margin of the card.
-/// @param {List} $border - Border around the card.
-/// @param {List} $radius - border radius of the card.
-/// @param {List} $shadow - box shadow of the card.
+/// @param {Color} $background [$white] - Background color of the card.
```

Comment:
For all of these params, I think we should either use the actual value or use the name of the variable actually used... as these stand, I'm very confused because they're using variables that are 1 level up the inheritance hierarchy.

@zurb/yetinauts do y'all have an opinion on actual value vs name? Looking around i see both but more the latter, which would mean these should be something more like

```
@param {Color} $background [$card-background] - ...
```